src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -668,8 +675,13 @@ static bool pick_from_internal_rr_locked(
 
 static grpc_lb_policy_args *lb_policy_args_create(grpc_exec_ctx *exec_ctx,
                                                   glb_lb_policy *glb_policy) {
-  grpc_lb_addresses *addresses =
-      process_serverlist_locked(exec_ctx, glb_policy->serverlist);
+  grpc_lb_addresses *addresses;
+  if (glb_policy->fallback) {
+    addresses = glb_policy->fallback_addresses;
```

Comment:
I see! I didn't notice that `addresses` was pointing to some local variable. I just focused on using the correct content.

Will fix it.